### DISCUSSION TITLE
Add A Dimension Mod Filter

### DISCUSSION THREAD

**Discussion by AwesomeSavage** [modrinth/.github#195] [Feature requests]
Something I liked about Curseforge (the inferior website) is it had a dimension mod category, which Modrinth currently does not have. It would be nice to be able to go down a list of dimension mods and download any ones that look cool, and dimensions are one of the core features of many mods, along with Minecraft itself.

**Comment by triphora** [comment]
https://github.com/orgs/modrinth/discussions/105
